modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -102,12 +102,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: Fork"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddForkNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddForkNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddForkNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Title");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Fork",
+			nodePosition = "-20,-40",
+			nodeType = "fork");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.

Reply:
Is there still a possibility to change? Yes, I missed that detail.
anything I can go back and put this step to the test to be more complete, ok

Reply:
@Italo-gabriel It's already been forwarded, there's nothing to worry about, but it would be interesting to check next time, thanks!